README.md

```diff
@@ -1,5 +1,3 @@
 # Loop Development Kit
 
 This is the central repository for all the Olive Help LDKs. Please see the individual LDK readmes in the `ldk\{language}` folder.
```

Comment:
This text should also be updated to refer to only one LDK and point to the JS LDK's folder specifically

Reply:
Yeah, makes sens